config/initializers/honeycomb.rb

```diff
@@ -1,4 +1,4 @@
-if Rails.env.test? || ApplicationConfig["HONEYCOMB_API_KEY"].blank?
+if Rails.env.test?
```

Comment:
I believe the original issue was the api key had been an empty string - as it is in my .env file (and in `.env_sample`) and the honeycomb client is looking for !key (which is false, since "" is truthy).

We may need to use `presence` on line 6 when setting the api key from env/appconfig to work around that (and coerce Libhoney::Client to use null transport instead of raising errors).

Reply:
yeah - I don't get any failures either and I'm probably being pedantic - I do see the client is not using nullclient or nulltransmission - I wonder if it's swallowing those errors further down.

I can confirm this smooths out the memory use after running a few tens of thousands of jobs and only seeing 40-50MB instead of 300+MB as before.